packages/thirdweb/src/react/web/ui/ConnectWallet/ConnectButtonProps.ts

```diff
@@ -98,8 +98,8 @@ export type ConnectButton_connectButtonOptions = {
    * For some CSS properties, you may need to use the `!important` to override the default styles
    *
    * ```tsx
-   * <ConnectButton button={{
-   *  className="my-custom-class"
+   * <ConnectButton connectButton={{
+   *   className="my-custom-class"
```

Comment:
this cannot be correct, if anything this has to be `className: "..."` please double check

Reply:
oops